src/SFA.DAS.AssessorService.Web.UnitTests/CertificateTests/CertificateVersionControllerTests.cs

```diff
@@ -34,6 +34,7 @@ public class CertificateVersionControllerTests
         private const int Ukprn = 123456;
         private const string Username = "TestProviderUsername";
         private Guid CertificateId = Guid.NewGuid();
+        private const string EpaoId = "EpaoId1234";
```

Comment:
Doesn't really matter but this would typically be like EPA0123 as that is the actual format

Reply:
Have updated the value to be a better reflection 👍